GoogleChromeLabs/ps-analysis-tool
# Feature: Add context menu to copy network filter string to filter network requests

## Description

In this PR for v0.4 we are allowing users to copy the network filter string to filter the network requests for each cookie in the network.

## Relevant Technical Choices
- Since clipboard API is not being allowed to be used in the extension we are using the `execCommand` to copy the network string to clipboard.

## Testing Instructions
- In the terminal run `npm start`
- Now go to any site and open Privacy Sandbox in the DevTools panel.
- Go to the table. Right-click on the row and in the context menu click on `Copy network filter string.`.
- Go to the network tab in the filter input paste you should see a string in the format `cookie-domain:${domain} cookie-name:${name}`.

## Additional Information:



## Screenshot/Screencast

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/59614577/93d4e306-6dd2-4de3-8baa-001ec2feb81c


---



Fixes #320
